From 500132a7671c4fd0fc89bb2579fb5692be77ea83 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 2 Mar 2017 00:59:09 +0800
Subject: [PATCH 1687/5242] MLK-14289 ARM: imx: add inline for unused function

commit  1613a8b7c81d9fac7797fc6121dbeaafddee88e2 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add inline for unused function to avoid build warning.

For example, when i.MX7D is unselected, below warning
will show during compile:

arch/arm/mach-imx/common.h:163:13: warning:
'imx_gpcv2_add_m4_wake_up_irq' defined but not used
[-Wunused-function]

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/common.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-imx/common.h b/arch/arm/mach-imx/common.h
index 32be13c..3380664 100644
--- a/arch/arm/mach-imx/common.h
+++ b/arch/arm/mach-imx/common.h
@@ -140,7 +140,7 @@ static inline void imx_smp_prepare(void) {}
 #else
 static inline int imx_gpcv2_mf_power_on(unsigned int irq, unsigned int on) { return 0; }
 static inline void imx_gpcv2_set_core1_pdn_pup_by_software(bool pdn) {}
-static void imx_gpcv2_add_m4_wake_up_irq(u32 hwirq, bool enable) {}
+static inline void imx_gpcv2_add_m4_wake_up_irq(u32 hwirq, bool enable) {}
 #endif
 void __init imx_gpcv2_check_dt(void);
 void imx_gpcv2_set_lpm_mode(enum mxc_cpu_pwr_mode mode);
-- 
1.7.9.5

